doc/cephadm/monitoring.rst

```diff
@@ -140,6 +140,88 @@ For example
 
           ceph config rm mgr mgr/cephadm/container_image_prometheus
 
+Using custom configuration files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By overriding cephadm templates, it is possible to completely customize the
+configuration files for monitoring services.
+
+Internally, cephadm already uses `Jinja2
```

Comment:
Singular? "A template"?
```suggestion
Internally, cephadm already uses `Jinja2
```

Reply:
There's one template for each service. I'd argue for templates.